SharedContent/cpp-winui-template/WinUI3TemplateCpp/SettingsPage.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<Page
+    x:Class="WinUI3TemplateCpp.SettingsPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinUI3TemplateCpp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid RowDefinitions="*,Auto">
+
+        <StackPanel HorizontalAlignment="Left" Spacing="20" >
```

Comment:
Should make an XML prettify pass over these XAML files to ensure consistency of indentation, attribute alignment, space-before-close-tag, etc.

Should also have a comment in the samples guide about what prettify parameters to use.

Reply:
Adding an `.editorconfig` file to the sln that captures the coding style you're following would be good too.

Reply:
Added `.editorconfig` and `prettierc` files to the repo